packages/tools/devtools/devtools-view/src/components/SettingsView.tsx

```diff
@@ -109,20 +109,31 @@ export function SettingsView(): React.ReactElement {
 	return (
 		<div className={styles.root}>
 			<div className={styles.section}>
-				<h4 className={styles.sectionHeader}>Theme</h4>
+				<h4 title="ThemeTitle" className={styles.sectionHeader}>
```

Comment:
Just out of curiosity, does including `title` in the HTML tags improve accessibility?

Reply:
It can but in this case, the title itself isn't navigable and thus not strictly needed for the test. 
Will remove.